plugins/terraform/resource_environment_link_test.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"errors"
+	"log"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func TestEnvironmentLink_defaults(t *testing.T) {
+	ctrl, mockClient, provider := setupUnitTest(t)
+	defer ctrl.Finish()
+
+	mockClient.EXPECT().
+		CreateLink("test-env", "test-env2").
+		Return(nil)
+
+	mockClient.EXPECT().
+		GetEnvironment("test-env").
+		Return(&models.Environment{}, nil)
+
+	environmentResource := provider.ResourcesMap["layer0_environment_link"]
+	d := schema.TestResourceDataRaw(t, environmentResource.Schema, map[string]interface{}{
+		"source": "test-env",
+		"dest":   "test-env2",
+	})
+
+	if err := environmentResource.Create(d, mockClient); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestEnvironmentLinkRead_noenv(t *testing.T) {
```

Comment:
I added this test, although it doesn't add anything to our coverage nor does it test anything that isn't already tested in `TestEnvironmentLinkCreate`